libkbfs/chat_local.go

```diff
@@ -144,6 +144,10 @@ func (c *chatLocal) GetConversationID(
 			continue
 		}
 
+		if cb == nil && config.KBFSOps() != nil {
+			cb = config.KBFSOps().NewNotificationChannel
+		}
```

Comment:
What if `cb` is still `nil` here?

Reply:
Then we'll panic!  But I'm pretty sure it's impossible for this function to be called in local mode without `KBFSOps()` being initialized.  If that ever happens though, the test will crash and we can fix it.  I'll leave it for now.

Reply:
Sounds good